v2/component_analyses.yaml

```diff
@@ -0,0 +1,288 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: "2"
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Fabric8 Server"
+security:
+- tokenAuth: []
+paths:
+  /component-analyses/{eco}/{name}/{ver}:
+    get:
+      tags:
+      - Scan Services
+      summary: Get security and other details about a software component and recommendations
+        to change to a more secured version
+      operationId: f8a_scanner.api_v2.component-analyses
+      parameters:
+      - name: eco
+        in: path
+        description: component ecosystem
+        required: true
+        schema:
+          type: string
+      - name: name
+        in: path
+        description: component name
+        required: true
+        schema:
+          type: string
+      - name: ver
+        in: path
+        description: component version
+        required: true
+        schema:
+          type: string
+      - name: security-vendor
+        in: header
+        description: Security Vendor
+        schema:
+          type: string
+          enum:
+          - snyk
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ComponentAnalysesResponse'
```

Comment:
Need to model two different responses: free tier and registered user personas

Reply:
Ack!

Reply:
Still I don't see the notion of Registered/Unregisterd users in model.

Reply:
@arajkumar I have added `PrivateRecommendation` and `PublicRecommendation` and means obvious.

Reply:
These field names can be confusing (both the personas can see part / complete set of private vuln. information). Please name these fields based on user persona: free-tier/registered user

Reply:
Please recheck if the contents under free-tier / registered user response models clearly indicate only the fields that are displayed in the UI.

Reply:
Ack!